.github/actions/devcontainer_run_command/action.yml

```diff
@@ -211,7 +211,7 @@ runs:
       shell: bash
       run: |
         # Write command to a command.sh script file
-        cat <<EOF > ./command.sh
+        cat <<EOF >> ./command.sh
```

Comment:
Why do we need this change?

We don't expect the file to exist. If it does, we don't want the command to be appended. Why is the original code wrong? Am I missing something?